module/Company/src/Company/Model/Job.php

```diff
@@ -233,24 +247,97 @@ public function setDescription($description)
     }
 
     /**
-     * Get the job's company.
-     *
-     * @return Company
+     * Get the job's language.
+     * 
+     * @return string language of the job
+     */
+    public function getLanguage()
+    {
+        return $this->language;
+    }
+
+    /**
+     * Set the job's language.
+     * 
+     * @param string $language language of the job
      */
-    public function getCompany()
+    public function setLanguage($language)
     {
-        return $this->company;
+        $this->language = $language;
     }
 
     /**
-     * Set the job's company.
+     * Get the job's packet.
      *
-     * @param Company company
+     * @return CompanyPacket
      */
-    public function setCompany($company)
+    public function getPacket()
     {
-        $this->company = $company;
+        return $this->packet;
     }
 
+    /**
+     * Set the job's packet.
+     * 
+     * @param CompanyPacket $packet the job's packet
+     */
+    public function setPacket(CompanyPacket $packet)
+    {
+        $this->packet = $packet;
+    }
+
+    // For zend2 forms
+    /**
+     * Returns an array containing all instance variables of this object
+     *
+     */
+    public function getArrayCopy()
+    {
+        $array = get_object_vars($this);
 
+        if ($this->getActive()) {
+            $array['active'] = '1';
+        } else {
+            $array['active'] = '0';
+        }
+
+        return $array;
+    }
+    
+    /**
+     * Returns the first argument if it is nonnul, else, returns the second 
+     * argument
+     *
+     * @param mixed $object
+     * @param mixed $default
+     */
+    private function updateIfSet($object, $default)
+    {
+        if (isset($object)) {
+            return $object;
+        }
+        return $default;
+    }
+    /**
+     * Sets all instance variables of this object to the values of the entries 
+     * in $data
+     *
+     * @param mixed $data
+     */
+    public function exchangeArray($data)
+    {
+        $this->name = $this->updateIfSet($data['name'],'');
+        $this->slugName = $this->updateIfSet($data['slugName'],'');
+        $this->language = $this->updateIfSet($data['language'],'');
+        $this->website = $this->updateIfSet($data['website'],'');
+        $this->email = $this->updateIfSet($data['email'],'');
+        $this->phone = $this->updateIfSet($data['phone'],'');
+        $this->description = $this->updateIfSet($data['description'],'');
+        $lActive = $data['active'];
+        if ($lActive === 1) {
+            $this->active = true;
+        } else {
```

Comment:
Else is considered relatively bad practise in Object Oriented code. If you don't really need it, use a combination with `return` instead.

Also, we can condense this into `$this->active = ($lActive === 1)`.